build/azure-pipelines/web/build/Dockerfile

```diff
@@ -8,7 +8,14 @@ RUN apt-get update && apt-get upgrade -y
 
 RUN apt-get install -y libxkbfile-dev pkg-config libsecret-1-dev libxss1 dbus xvfb libgtk-3-0 \
 	libkrb5-dev git apt-transport-https ca-certificates curl gnupg-agent software-properties-common \
-	libnss3 libasound2 make gcc libx11-dev fakeroot rpm libgconf-2-4 libunwind8 g++ python
+	libnss3 libasound2 make gcc libx11-dev fakeroot rpm libgconf-2-4 libunwind8 g++ python wget
+
+# Set the Chrome repo.
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
```

Comment:
not really a big deal, but from link 21 it looks like curl is already installed.  It probably could be used interchangeably with wget most the time.

Reply:
I will look into installing with curl, so we can get wget out of there.

Reply:
It's up to you.  If for some reason curl doesn't just work, I wouldn't worry about it.